youtube_dl/extractor/npo.py

```diff
@@ -406,6 +406,32 @@ def _real_extract(self, url):
         }
 
 
+class SchoolTVIE(InfoExtractor):
+    IE_NAME = 'schooltv'
+    _VALID_URL = r'https?://(?:www\.)?schooltv\.nl/video/(?P<id>.*[^/])/?'
+
+    _TEST = {
+        'url': 'http://www.schooltv.nl/video/ademhaling-de-hele-dag-haal-je-adem-maar-wat-gebeurt-er-dan-eigenlijk-in-je-lichaam/',
+        'info_dict': {
+            'id': 'WO_NTR_429477',
+            'title': 'Ademhaling: De hele dag haal je adem. Maar wat gebeurt er dan eigenlijk in je lichaam?',
+            'ext': 'm3u8',
+            'description': 'md5:abfa0ff690adb73fd0297fd033aaa631'
+        }
+    }
+
+    def _real_extract(self, url):
+        playlist_id = self._match_id(url)
+        webpage = self._download_webpage(url, playlist_id)
+
+        entries = [
+            self.url_result('npo:%s' % video_id)
+            for video_id in re.findall(r'VideoPlayer.*data-mid="([^"]+)"', webpage)
+        ]
+        playlist_title = self._og_search_title(webpage)
+        return self.playlist_result(entries, playlist_id, playlist_title)
```

Comment:
From what I've seen there is always only one video on the page thus no need in playlist.